pkgs/top-level/all-packages.nix

```diff
@@ -16497,7 +16497,8 @@ with pkgs;
     electron_18
     electron_19
     electron_20
-    electron_21;
+    electron_21
+    electron_22;
```

Comment:
I am not an Electron user; however, does this major update  break anything? Or is it backwards-compatible?

Reply:
We've been always doing this. Most of the packages are using pinned versions of the package, not a generic alias.